src/main/java/org/spongepowered/api/entity/living/Tameable.java

```diff
@@ -26,11 +26,12 @@
 
 import com.google.common.base.Optional;
 import org.spongepowered.api.entity.Tamer;
+import org.spongepowered.api.entity.living.animal.Animal;
 
 /**
- * Represents an Agent that can be tamed by a Tamer.
+ * Represents an Animal that can be tamed by a Tamer.
  */
-public interface Tameable extends Agent {
+public interface Tameable extends Animal {
```

Comment:
I made absolutely sure that it isn't necessary to have Tameable extend Animal. Please stop this nonsense of re-engineering the entity structure to match Minecraft's entity structure. There isn't a reason for any mod to add a creeper that is also a Tameable creeper. 

Otherwise I don't see where this entire PR is "urgent"